### ISSUE TITLE
Cannot find _ (underscore character) Ethos 1.0.11 RC4

### ISSUE THREAD

**Issue by spadeadam** [FrSkyRC/ETHOS-Feedback-Community#927]
X10 Horus Express 

I have downloaded latest 1.0.11 RC4 but cannot find underscore character (_) when naming models etc

Am I missing something easy?

**Comment by GiorgioCirelli** [comment]
You can find it under the numbers

![image](https://user-images.githubusercontent.com/42900080/132568578-a2fc6dfd-7e78-4922-9c2a-8606d2fdeb68.png)

**Comment by spadeadam** [comment]
> You can find it under the numbers
> 
> ![image](https://user-images.githubusercontent.com/42900080/132568578-a2fc6dfd-7e78-4922-9c2a-8606d2fdeb68.png)

Hi Giorgio, thank you for your reply.
I can edit the Name by turning the right knob but I cannot get the numbers screen to display with the underscore in it?

**Comment by spadeadam** [comment]
Hi Giorgio, many thanks for your reply, can you tell me how to get the numbers screen to display please

 

Many thanks

**Comment by GiorgioCirelli** [comment]
In the "Edit Model" screen click? 123.
Hello

![image](https://user-images.githubusercontent.com/42900080/132690616-425af1e2-2e0e-4ed6-ae48-d0aa8e494af7.png)

**Comment by spadeadam** [comment]
Hi Giorgio,

Many thanks for your help and patience, I am getting the Edit Model screen with Name, Picture, Model Type, Ailerons etc but I am not getting the Keyboard Overlay screen which displays the Numbers and letters, do I need to press a key to get the keyboard up?

 

The only way I can type text and numbers is with the scroll key where your screen shows Extra 1

 

MTIA, Mark

**Comment by GiorgioCirelli** [comment]
With the firmware version Release 1.0.11 RC 4 should open automatically by clicking on the model name. Try to upgrade again

**Comment by spadeadam** [comment]
Hi Giorgio,

 

Again many thanks for your help, I have downloaded 10X Express 1.0.11RC4 and Horus SD again and reinstalled.

 

I have created a new model and when I try and input the model name I have the same problem (no underscore can be found and no overlaid keyboard or numbers).

 

As all my other radios and TX’s (on opentx) have an underscore in the registered ID I do not want to change them all because Ethos cannot create an underscore (in my case) so I think I will change my X10 Express back to Opentx so that everything works again.

 

I will maybe try Ethos again in a year or two when they have sorted some of the bugs out and have a stable system

 

Many thanks for your help!

**Comment by MIKE-DAILEY** [comment]
How did you plan to use the touch screen keyboard for the Tandem systems on the X10 Express without a touch screen?

**Comment by spadeadam** [comment]
Hi, many thanks for your input, I raised an issue on the Ethos feedback site because I could not find the underscore character which I require to register all my existing receivers (which talk to my 2 x X9D 2019 Tx’s)
Giorgio kindly sent me images of X10 Express screen showing the pop-up keyboard (the screen images sent are not X20). I realise that the X10 Express does not have a touch screen hence me raising an issue on Github which maybe an oversight or bug which does not display the underscore character (if that makes sense).

MTIA

**Comment by MIKE-DAILEY** [comment]
There is no virtual keyboard for the Horus systems.  This issue merged off track and is now closed.  You might want to open a new issues and simply request that "underscore" character be added to the Horus ETHOS text edit function.

**Comment by spadeadam** [comment]
Hi, thank you for your message, 

I already opened a request for the missing underscore character stating that I had a Horus X10 Express with firmware 1.0.11RC4 but it was closed within hours, presumably because users with X20 tandems, with the pop-up keyboard, could see it.

I am not sure that opening a request again will get me any further.

I am a bit mystified why OpenTx has the underscore character  but the new Ethos lacks it, surely we do not want to go backwards?

 

Again, many thanks for your input
